slime.el

```diff
@@ -195,27 +197,32 @@ This applies to the *inferior-lisp* buffer and the network connections."
 This name is interpreted relative to the directory containing
 slime.el, but could also be set to an absolute filename."
   :type 'string
+  :risky t
   :group 'slime-lisp)
 
 (defcustom slime-connected-hook nil
   "List of functions to call when SLIME connects to Lisp."
   :type 'hook
+  :risky t
   :group 'slime-lisp)
 
 (defcustom slime-enable-evaluate-in-emacs nil
   "*If non-nil, the inferior Lisp can evaluate arbitrary forms in Emacs.
 The default is nil, as this feature can be a security risk."
   :type '(boolean)
+  :risky t
   :group 'slime-lisp)
 
 (defcustom slime-lisp-host "127.0.0.1"
   "The default hostname (or IP address) to connect to."
   :type 'string
+  :risky t
   :group 'slime-lisp)
 
-(defcustom slime-port 4005
+(defcustom slime-port "~/.slime/socket"
```

Comment:
This probably works; notably what emacs' server functionality does is creates a socket file at `/tmp/{username}{userid}/server`, which is another way to go